test/differentials/composite.jl

```diff
@@ -219,7 +219,10 @@ end
 
     @testset "show" begin
         @test repr(Composite{Foo}(x=1,)) == "Composite{Foo}(x = 1,)"
-        @test repr(Composite{Tuple{Int,Int}}(1, 2)) == "Composite{Tuple{Int64,Int64}}(1, 2)"
+        @test occursin(
```

Comment:
A comment here explaining what you're testing for would be helpful as my ability to parse regex is _very_ limited.